packages/osd-plugin-helpers/README.md

```diff
@@ -37,37 +38,44 @@ $ plugin-helpers help
   Usage: plugin-helpers [command] [options]
 
   Commands:
-      build
-        Copies files from the source into a zip archive that can be distributed for
-        installation into production OpenSearch Dashboards installs. The archive includes the non-
-        development npm dependencies and builds itself using raw files in the source
-        directory so make sure they are clean/up to date. The resulting archive can
-        be found at:
+    build
+      Copies files from the source into a zip archive that can be distributed for
+      installation into production OpenSearch Dashboards installs. The archive includes the non-
+      development npm dependencies and builds itself using raw files in the source
+      directory so make sure they are clean/up to date. The resulting archive can
+      be found at:
+
+        build/{plugin.id}-{opensearchDashboardsVersion}.zip
+
+      Options:
+        --skip-archive                       Don't create the zip file, just create the build/opensearch-dashboards directory
+        --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the built plugin will target
+     
+    version
+      Updates the versions in opensearch_dashboards.json and package.json files to the value provided or syncs it with the version of OpenSearch Dashboards
   
-          build/{plugin.id}-{opensearchDashboardsVersion}.zip
-  
-        Options:
-          --skip-archive        Don't create the zip file, just create the build/opensearch-dashboards directory
-          --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the
-    
+      Options:
+        --sync        Update the versions to match Dashboards' (default)
+        --set         Update the version to a specific semantic version value
```

Comment:
I agree with josh, a simple example here will definitely be more helpful. semantic is well understood but but given our x.x.x.x versioning system it may be a little confusing as to what we expect as the semantic version. Also its a lot easier with examples

Reply:
I added some text.